flowmachine/flowmachine/features/utilities/subscriber_locations.py

```diff
@@ -79,7 +80,7 @@ def _make_query(self):
         return sql
 
 
-class BaseLocation(Query):
+class BaseLocation(Query, metaclass=ABCMeta):
```

Comment:
What's the point in making this an abstract base class if it doesn't contain any abstract methods?

Reply:
Because it doesn’t define a column_names prop. But, looking at it again there’s no real reason for it to inherit from Query - could just be used as a mixin.